recipes/libsodium/all/conanfile.py

```diff
@@ -4,7 +4,7 @@
 from conan.tools.files import export_conandata_patches, apply_conandata_patches, get, rmdir, copy, rm, replace_in_file
 from conan.tools.gnu import Autotools, AutotoolsToolchain
 from conan.tools.layout import basic_layout
-from conan.tools.microsoft import is_msvc, is_msvc_static_runtime, MSBuildDeps, MSBuildToolchain, MSBuild, VCVars, unix_path, msvc_runtime_flag, vs_layout
+from conan.tools.microsoft import is_msvc, is_msvc_static_runtime, MSBuildDeps, MSBuildToolchain, MSBuild, VCVars, unix_path, vs_layout
 import os
 
 required_conan_version = ">=1.52.0"
```

Comment:
Since this adds `rm_Safe`

```suggestion
required_conan_version = ">=1.53.0"
```